src/urllib3/util/ssl_.py

```diff
@@ -33,32 +35,35 @@ def _is_ge_openssl_v1_1_1(
     )
 
 
-try:  # Do we have ssl at all?
+if TYPE_CHECKING:
```

Comment:
This `if` statement can be independent from the `try-except` below I believe? Basically we still want the `from ssl import ...` to happen even when type checking